tests/conftest.py

```diff
@@ -41,3 +36,36 @@ def create_mock_response(mocker, response_data):
     response = mocker.MagicMock(spec=Response)
     response.text = json.dumps(response_data)
     return Py42Response(response)
+
+
+def assert_success(connector):
```

Comment:
These are some utility functions that make writing the tests a bit more concise. One thing I tried to always do is have each test check the status was successful (or error, if that was what it was supposed to be)